lib/common/Makefile

```diff
@@ -0,0 +1,12 @@
+CC ?= gcc
```

Comment:
@pinealservo - yes it does cause issues for us at present, but I think we should go ahead and merge anyway. We can discuss the correct approach separately. We have the option of maintaining our own repo/patches/branches....